next.config.js

```diff
@@ -1263,7 +1263,71 @@ const nextConfig = {
                 destination: '/concepts/v2/concepts/templates-and-roles/overview',
                 permanent: true
             },
-
+            {
+                source: '/docs/server-side/v2/Rooms/:slug',
+                destination: '/server-side/v2/api-reference/Rooms/:slug',
+                permanent: true
+            },
+            {
+                source: '/server-side/v2/active-rooms/:slug',
+                destination: '/server-side/v2/api-reference/active-rooms/:slug',
+                permanent: true
+            },
+            {
+                source: '/server-side/v2/Sessions/:slug',
+                destination: '/server-side/v2/api-reference/Sessions/:slug',
+                permanent: true
+            },
+            {
+                source: '/server-side/v2/policy/:slug',
+                destination: '/server-side/v2/api-reference/policy/:slug',
+                permanent: true
+            },
+            {
+                source: '/server-side/v2/Destinations/:slug',
+                destination: '/server-side/v2/api-reference/Destinations/:slug',
+                permanent: true
+            },
+            // {
+            //     source: '/server-side/v2/legacy-api%20(v1)/:slug',
```

Comment:
getting an error like this, working with @KaustubhKumar05 to fix it. 
![Screenshot 2023-02-20 at 1 02 51 PM](https://user-images.githubusercontent.com/105427758/220042407-406dd42b-ccec-448d-bf10-c5792e0d6733.png)